components/brave_wallet/browser/eth_tx_manager_unittest.cc

```diff
@@ -942,8 +945,11 @@ TEST_F(EthTxManagerUnitTest, ProcessHardwareSignatureFail) {
   callback_called = false;
   eth_tx_manager()->ProcessHardwareSignature(
       tx_meta_id, "0x00", "9ff044f89c205dd76a194f8b11f50d2eade744e", "",
-      base::BindLambdaForTesting([&](bool success) {
+      base::BindLambdaForTesting([&](bool success,
+                                     mojom::ProviderError error_out,
+                                     const std::string& error_message_out) {
         EXPECT_FALSE(success);
+        EXPECT_EQ(error_out, mojom::ProviderError::kInternalError);
```

Comment:
check if error message match the specific msg, or at least make sure it's not empty.